### PR TITLE
added scenario for bugzilla_1655130

### PR DESCRIPTION
**Pull request by omkarkhatavkar**
### PR Description
An added scenario for Bugzilla https://bugzilla.redhat.com/show_bug.cgi?id=1655130
### Dependancy 
Needs Airgun PR  https://github.com/SatelliteQE/airgun/pull/420

### CODE REVIEW

File: tests/foreman/ui/test_errata.py [hunk 1]
```diff
@@ -525,6 +527,49 @@ def test_positive_content_host_search_type(session, erratatype_vm):
...
+
+    :id: 236de56f-8035-4072-b0fa-03abbf3fc692
+
+    :Steps:
+
+        1. Content Host with applicable security errata.
+        2. Create a User with 'Viewer' Permission.
+        3. Go to Content Hosts -> Select content host -> Errata Tab -> Select Errata_Id.
+
+    :expectedresults: The errata details apge should be disaplyed for User.
```
Comment (vijay8451) [inline tests/foreman/ui/test_errata.py]:
```suggestion
    :expectedresults: The errata details page should be displayed for User.
```

File: tests/foreman/ui/test_errata.py [hunk 2]
```diff
@@ -525,6 +527,49 @@ def test_positive_content_host_search_type(session, erratatype_vm):
...
+        query={'search': u'name="Viewer"'})
+    default_loc_id = entities.Location().search(
+         query={'search': 'name="{}"'.format(DEFAULT_LOC)})[0].id
+    default_loc = entities.Location(id=default_loc_id).read()
+
+    entities.User(role=viewer_role, login=login, password=password,
+                  default_location=default_loc, organization=[module_org]).create()
+
+    pkgs = " ".join(FAKE_9_YUM_OUTDATED_PACKAGES)
+    assert _install_client_package(erratatype_vm, pkgs, errata_applicability=True)
```
Comment (vijay8451) [inline tests/foreman/ui/test_errata.py]: I see the `pkgs` in only using at one place ..Can we use as below it :
```
assert _install_client_package(erratatype_vm, 
                               " ".join(FAKE_9_YUM_OUTDATED_PACKAGES), 
                               errata_applicability=True)
```

File: tests/foreman/ui/test_errata.py [hunk 3]
```diff
@@ -525,6 +527,49 @@ def test_positive_content_host_search_type(session, erratatype_vm):
...
+    :BZ: 1655130
+
+    :CaseLevel: Integration
+    """
+    login = gen_string('alpha')
+    password = gen_string('alpha')
+    viewer_role = entities.Role().search(
+        query={'search': u'name="Viewer"'})
+    default_loc_id = entities.Location().search(
+         query={'search': 'name="{}"'.format(DEFAULT_LOC)})[0].id
```
Comment (vijay8451) [inline tests/foreman/ui/test_errata.py]: Any specific reason to use `DEFAULT_LOC`? Can we not create a new one here?

Comment (omkarkhatavkar) [reply tests/foreman/ui/test_errata.py]: avoid increase time and this is a UI test we are seeing issues where we are selecting the location causing the satellite to take time load. I have seen even other tests most of them have used the default location. 

What are the befits are you seeing adding the different location?

File: tests/foreman/ui/test_errata.py (hunk 2 above)
Comment (omkarkhatavkar) [reply tests/foreman/ui/test_errata.py]: In your above example, we are using three lines of code where an existing example uses the 2 lines of code. At looking at cleanness I think the current solution looks better.

File: tests/foreman/ui/test_errata.py (hunk 3 above)
Comment (vijay8451) [reply tests/foreman/ui/test_errata.py]: Benefits are as similar to create a new org ... but it was a suggestion only ... 
as you mentioned it's already using (which I think not a right way) at mutiple place we can ignore it here and can handle under separate PR ... thanks

File: tests/foreman/ui/test_errata.py (hunk 2 above)
Comment (vijay8451) [reply tests/foreman/ui/test_errata.py]: It was a suggestion only :) .. thanks

**Pull request merged**